bcda/cclf/testutils/cclfUtils.go

```diff
@@ -43,7 +44,7 @@ func ImportCCLFPackage(acoSize, environment string) (err error) {
 		return errors.New("invalid argument for environment")
 	}
 
-	sourcedir := filepath.Join("shared_files/syntheticCCLFFiles", environment, acoSize)
+	sourcedir := filepath.Join("shared_files/cclf/files/synthetic", environment, acoSize)
```

Comment:
I struggled with that and am definitely open to suggestions! I ended up calling the directory `synthetic` because that kept it closest to its old path name. It contains the files used for the `import-synthetic-cclf-package` command, so maybe something to do with that?

Reply:
yeah I have no real suggestions..synthetic is fine :)